### DISCUSSION TITLE
How to add Shadcn UI to an exsting turbo repo Nextjs project?

### DISCUSSION THREAD

**Discussion by alamenai** [shadcn-ui/ui#6139] [Q&A]
Hi,

I have a mono repo project and I want to add Shadcn to `packages/ui` , I tried to follow up [on these steps](https://ui.shadcn.com/docs/monorepo) but they refer only to scaffolding a new project from scratch.

When I tried this command:
```
npx shadcn@canary init
```

it just added Shadcn to the dedicated web app but not as mentioned in the documentation.

**Comment by FADHILI-Josue** [comment]
@alamenai, You can check out this [link](https://matinkhani.medium.com/create-a-turborepo-with-nextjs-tailwindcss-shadcn-6e6ecfd52aea) for a guide on setting up TailwindCSS and Shadcn UI in TurboRepo with Next.js. It should help you get everything up and running smoothly!

**Comment by bernard-ng** [comment]
Here is a comprehensive guide : https://ngandu.hashnode.dev/monorepo-nextjs-shadcnui-bun
